operations/loyaltyprograms.md

```diff
@@ -0,0 +1,88 @@
+# Loyalty programs
+
+## Get all loyalty programs
+
+Returns all loyalty programs of the enterprise, possibly filtered by identifiers.
```

Comment:
- Can I suggest slightly different wording "Returns all loyalty programs of the enterprise (in the given activity state), optionally filtered by specific loyalty program identifiers or other filter parameters."
- Oh, and I would also add the sentence "Note this operation uses [Pagination](../guidelines/pagination.md)"